src/class-parsely.php

```diff
@@ -359,6 +359,16 @@ public function get_options(): array {
 			return $this->option_defaults;
 		}
 
+		/**
+		 * Makes the usage of `apikey` backward compatible while replacing it with `site_id`.
+		 *
+		 * @since 3.7.0.
+		 */
+		if ( isset( $options['apikey'] ) ) {
+			$options['site_id'] = $options['apikey'];
+			unset( $options['apikey'] );
```

Comment:
Could this result in removal of the `apikey` setting from the database?

If yes, let's not remove the `apikey` option yet, in case a plugin downgrade is needed for any reason. We could add a comment for this to remove it in wp-parsely version 4 for example.